python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_p2p_links.schema.yml

```
@@ -189,6 +189,22 @@ $defs:
               type: str
               valid_values: ["on", "active", "passive"]
               default: active
+            channel_id_algorithm:
+              type: str
+              valid_values:
+                - "first_port"
+                - "p2p_link_id"
+              default: first_port
+              description: |-
+                  Configures how to generate the port-channel ID.
+                  By default the ID is generated based on the first switch port from node_child_interfaces[].interfaces.
+                  The `p2p_link_id` setting will use the `id` for each link added with the `channel_id_offset` to determine the port-channel ID.
+            channel_id_offset:
```

We need to add a description to explain when this offset applies. Otherwise they may expect it to also apply in the first_port case.
```suggestion
            channel_id_offset:
              description: Offset added to the ID of the p2p_link when `channel_id_algorithm` is set to `p2p_link_id` and `channel_id` is not set directly.
```